Source/SelfService/Web/layout/layout.scss

```diff
@@ -30,18 +31,22 @@ svg.logo {
 
   .sidebar ul {
     list-style-type: none;
+    padding: 0;
+    margin: 0;
```

Comment:
I think it makes more sense to move this into the `sx`-attribute of the component, now that we have this possibility. The only caveat is if it causes a FOUC (Flash of unstyled content) by moving it in.